registry/remote/repository.go

```diff
@@ -182,7 +182,7 @@ func (r *Repository) PushTag(ctx context.Context, expected ocispec.Descriptor, c
 func (r *Repository) push(ctx context.Context, expected ocispec.Descriptor, content io.Reader, reference string) error {
 	ref := r.Reference
 	ref.Reference = reference
-	ctx = withScopeHint(ctx, ref, auth.ActionPush)
+	ctx = withScopeHint(ctx, ref, auth.ActionPull, auth.ActionPush)
```

Comment:
can you add some comments for the required scope? It will help future maintainer.

Reply:
documentation added